iphone/Classes/TiApp.m

```diff
@@ -797,33 +797,64 @@ - (void)URLSession:(NSURLSession *)session task:(NSURLSessionTask *)task didSend
     [[NSNotificationCenter defaultCenter] postNotificationName:kTiURLUploadProgress object:self userInfo:dict];
 }
 
+-(void)URLSession:(NSURLSession *)session dataTask:(NSURLSessionDataTask *)dataTask didReceiveData:(NSData *)data
+{
+    if(!nsURLUploadTaskResponses){
+        nsURLUploadTaskResponses = [[NSMutableDictionary alloc] init];
```

Comment:
How can this work? The property is named `uploadTaskResponses `. Please validate this again and rename + format properly.